src/helpers/getDate.ts

```diff
@@ -20,18 +20,21 @@ export const getMinDate = (date: Date | null): Date => {
         return addMinutes(currentDate, ADD_FIVE_MINUTES);
     }
 
+    const currentDayStart = startOfDay(currentDate);
+    const dateDayStart = startOfDay(date);
+
     if (isSameDay(date, currentDate)) {
-        return date > addMinutes(currentDate, ADD_FIVE_MINUTES) ? addMinutes(currentDate, ADD_FIVE_MINUTES) : date;
+        return addMinutes(currentDate, ADD_FIVE_MINUTES);
     }
 
-    if (date > currentDate) {
+    if (dateDayStart > currentDayStart) {
         return startOfDay(date);
     }
 
     return addMinutes(currentDate, ADD_FIVE_MINUTES);
 }
 
 export const getMaxDate = (date: Date) => {
-  const nextDay = addDays(date, ADD_ONE_DAY);
-  return endOfDay(nextDay);
+    const nextDay = addDays(date, ADD_ONE_DAY);
```

Comment:
Is ADD_ONE_DAY a good name for const ?
What if you need to use the same const for subtraction, multiplication, division ?